docs/literature.md

```diff
@@ -6,11 +6,13 @@ Collection of literature regarding scRNAseq.
 
 Literature related to general information around scRNAseq.
 
-### UMIs - Problem, solution and proof
+### UMIs - Problem, Solution and Proof
 
-A nice blog about UMIs from Tom Smith: [Visit Blog](https://cgatoxford.wordpress.com/2015/08/14/unique-molecular-identifiers-the-problem-the-solution-and-the-proof/)
+A nice blog about UMIs from Tom Smith: [Visit Blog](https://cgatoxford.wordpress.com/2015/08/14/unique-molecular-identifiers-the-problem-the-solution-and-the-proof/).
 
+### scRNA-Seq Analysis Tutorial
 
+A nice tutorial about how one could  do scRNA-Seq analysis can be found here: https://hemberg-lab.github.io/scRNA.seq.course/index.html.
```

Comment:
Written by the very same @wikiselev who wants to collaborate on this pipeline 😉

Reply:
Simon, this goes to our epic list ... :)